packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -256,8 +256,8 @@ const Dropdown = ({
 
     React.useEffect(() => {
         if (!initial_render.current && !is_list_visible && value) dropdown_ref.current.focus();
-    }, [is_list_visible, value]);
-
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_list_visible]);
```

Comment:
@likhith-deriv @aizad-deriv are you sure this change won't affect all dropdowns across the app?🙂